### PR TITLE
Mobile stack intros

### PR DESCRIPTION
**Pull request by i-like-robots**
Improves place stack intro on small screens:

- Full width
- Smaller text
- Biggerer button
- No 'inline' read more button

screen shot 2015-02-19 at 12 59 50

Waldorf PR: https://github.com/lonelyplanet/waldorf/pull/1817

Also fixes https://trello.com/c/tLSkiWvE/671-camera-icon-merged-overlayed-with-stack-titles

### CODE REVIEW

**Comment by Ianfeather** [comment]
Can we hide the usabilla thing on very small screens?

**Comment by i-like-robots** [comment]
I guess we should simply not load it at all, we'll probably need permission to touch it.

**Comment by stevebrownlee** [comment]
Hm, I thought Brad P already didn't have it being served on mobile

**Comment by i-like-robots** [comment]
It doesn't load on my phone but _does_ when emulating my phone in Chrome. Handy.

**Comment by stevebrownlee** [comment]
I'm not sure Usabilla's admin has an option to not load based on device size.  I believe it's user agent based, so I think we'd have to hack that functionality.

File: app/assets/stylesheets/core/core_components/_card_page.sass [hunk 1]
```diff
@@ -160,3 +165,19 @@ $card--page-padding-x: 25px
 
   .card--page__content + &
     border-top: 1px solid $body-background
+
+// -----------------------------------------------------------------------------
+//  Variations
+// -----------------------------------------------------------------------------
+
+// <https://trello.com/c/szd8dMed/>
```
Comment (remybach) [inline app/assets/stylesheets/core/core_components/_card_page.sass]: I'm not sure a link to a Trello card is very useful... a short description might make more sense here.

**Pull request merged**